R/qenv-constructor.R

```diff
@@ -10,13 +10,11 @@
 #' `qenv()` instantiates a `qenv` with an empty environment.
 #' Any changes must be made by evaluating code in it with `eval_code` or `within`, thereby ensuring reproducibility.
 #'
-#' `new_qenv()` (`r badge("deprecated")` and not recommended)
-#' can instantiate a `qenv` object with data in the environment and code registered.
-#'
 #' @name qenv
 #'
-#' @return `qenv` and `new_qenv` return a `qenv` object.
+#' @return `qenv` returns a `qenv` object.
 #'
+#' @seealso [`base::within()`], [`get_var()`], [`get_env()`], [`get_warnings()`], [`join()`], [`concat()`]
```

Comment:
I'm not sure about it. Why not `get_code`, `eval_code` : ) ?
And if you want to list them all, what's the point? There is `index` down below at the end of the help page, and we also have a vignette with the full documentation of `qenv` usage. I think in this case this `@seealso` is redundant. Please remove or please convince me it has any benefit.

Reply:
These are just the ones there were on the original help page I just moved them around. I didn't want to increase the scope of the PR. But I can delete this if you want, but I don't think it hurts to have more links to other functions from the package that help the user to know how to use the class created using that document.

`eval_code()` and `get_code()` are documented on this same help page, so it doesn't make sense to have a link to the same page.

Reply:
Alrighty. This make sense! I am not sure about `base::within` though

Reply:
What are your thoughts about `base::within` @llrs-roche ?

Reply:
alrighty